tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor.Tests/Static/IssueProcessingTests.cs

```diff
@@ -26,14 +26,15 @@ public class IssueProcessingTests : ProcessingTestBase
         // 6. RuleState ruleStateServiceAttention - Whether or not the ServiceAttention rule is on or off
         //
         // CODEOWNERS changes necessary to test
-        // The CODEOWNERS file needs entries that match the AIReturnLabels, in theory the AI Label Service will return one
+        // The CODEOWNERS file needs entries that match the AIReturnLabels, in theory the AI Issue Triage will return one
```

Comment:
```suggestion
        // The CODEOWNERS file needs entries that match the AIReturnLabels, in theory the AI Issue Triage service will return one
```

nit:  We're inconsistent in using "service" vs. "Service" vs. "" in these.   Doesn't matter which we choose, but it would be nice to align them.